src/mgr/MgrClient.cc

```diff
@@ -286,11 +280,6 @@ int MgrClient::start_command(const vector<string>& cmd, const bufferlist& inbl,
 
   ldout(cct, 20) << "cmd: " << cmd << dendl;
 
-  if (!session) {
-    lderr(cct) << "no session, waiting" << dendl;
-    wait_on_list(waiting_for_session);
-  }
```

Comment:
What queues them up for later sending? In this function we're calling send_message() if there's a session->con and otherwise just returning. I don't see anything in Monitor::handle_command() either.